### PR TITLE
Add 895.MaximumFrequencyStack.cpp

### PR DESCRIPTION
**Pull request by AhmadShaaban**
### What have you Changed(must)

Adding New Code

### Issue no.(must) - #1042

Pr will be close and marked as spam. If issue number not found or issue was assigned to someone else. 
Marking as spam can block your account from HacktoberFest.
### Self Check(Tick After Making pull Request)

- [x] This issue was assigned to me if not it will be marked as spam.
- [x] My file is in the proper folder
- [x] I am following clean code and Documentation.
- [x] I have added the title and what program will do first in the file
- [x] I have named the file as (Problem #). (Problem Title(in Camel Case)).(extension)
      ex. 2. AddToNumbers.py

### CODE REVIEW

File: C++/895.MaximumFrequencyStack.cpp [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+int main() {
+
+
+    ios::sync_with_stdio(false);
+    cin.tie(0);
+    cout.tie(0);
+
+    FreqStack* obj = new FreqStack();
+    obj->push(5);
+    cout << obj->pop()<<endl;
```
Comment (JOKUE2002) [inline C++/895.MaximumFrequencyStack.cpp:59]: This is not requested in the original challenge (https://leetcode.com/problems/maximum-frequency-stack/)

Please remove this.

File: C++/895.MaximumFrequencyStack.cpp [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+//FreqStack has two functions:
+//
+//push(int x), which pushes an integer x onto the stack.
+//pop(), which removes and returns the most frequent element in the stack.
+//If there is a tie for most frequent element, the element closest to the top of the stack is removed and returned.
+
+
+#include <bits/stdc++.h>
+
+using namespace std;
```
Comment (JOKUE2002) [inline C++/895.MaximumFrequencyStack.cpp:13]: This is not requested in the original challenge (https://leetcode.com/problems/maximum-frequency-stack/)

Please remove this.

**Review by JOKUE2002 — CHANGES_REQUESTED** [review]
Please change the Filename to include a space (' ') between the number and title (e.g. "895. MaximumFrequencyStack.cpp").

Please also do the changes requested in the comments (lines 11-13 and 50-59).

**Comment by AhmadShaaban** [comment]
@JOKUE2002 I have done the requested changes. can you check?

**Review by JOKUE2002 — APPROVED** [review]

**Pull request merged**